webapp/channels/src/components/markdown_preview.scss

```
@@ -0,0 +1,81 @@
+            text-decoration: none;
+
+            &:hover,
+            &:focus {
+                text-decoration: underline;
+            }
+        }
+
+        img {
+            max-width: 100%;
+            max-height: none;
+            height: auto;
```

Stylelint (17.14.0)

[error] Expected "height" to come before "max-height" (order/properties-order)

(order/properties-order)